src/components/Typography/style.css

```diff
@@ -0,0 +1,118 @@
+.gutter-bottom {
+  margin-bottom: 0.35em !important;
+}
+
+.typography-header,
+.typography-body {
+  padding: 0 !important;
+  margin: 0 !important;
+  color: #212121;
+  font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
+}
+
+.typography-h1 {
+  font-size: 6rem;
+  font-weight: 300;
+  line-height: 1.167;
+  letter-spacing: -0.01562em;
+}
+
+.typography-h2 {
+  font-size: 3.75rem;
+  font-weight: 300;
+  line-height: 1.2;
+  letter-spacing: -0.00833em;
+}
+
+.typography-h3 {
+  font-size: 3rem;
+  font-weight: 400;
+  line-height: 1.167;
+  letter-spacing: 0em;
+}
+
+.typography-h4 {
+  font-size: 2.125rem;
+  font-weight: 400;
+  line-height: 1.235;
+  letter-spacing: 0.00735em;
+}
+
+.typography-h5 {
+  font-size: 1.5rem;
+  font-weight: 400;
+  line-height: 1.334;
+  letter-spacing: 0em;
+}
+
+.typography-h6 {
+  font-size: 1.25rem;
+  font-weight: 500;
+  line-height: 1.6;
+  letter-spacing: 0.0075em;
+}
+
+.muted {
+  color: rgba(0, 0, 0, 0.54);
+}
+
+.typography-subtitle {
+  font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
+}
+.subtitle1 {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.75;
+  padding: 0 !important;
+  margin: 0 !important;
+  letter-spacing: 0.00938em;
+}
+.subtitle2 {
+  font-size: 0.875rem;
+  font-weight: 500;
+  line-height: 1.57;
+  letter-spacing: 0.00714em;
+}
+
+/* alignments */
+.typography-align-center {
+  text-align: center !important;
+}
+
+.typography-align-left {
+  text-align: left !important;
+}
+
+.typography-align-right {
+  text-align: right !important;
+}
+
+.typography-align-justify {
+  text-align: justify !important;
+}
+
+.typography-body1 {
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  letter-spacing: 0.00938em;
+}
+
+.typography-body2 {
+  font-size: 0.875rem;
+  font-weight: 400;
+  line-height: 1.43;
+  letter-spacing: 0.01071em;
+}
+
+.typography-primary {
+  color: #1976d2 !important;
+}
+
+.typography-secondary {
+  color: #dc004e !important;
+}
+
+.typography-white {
+  color: #fff !important;
```

Comment:
The colors which are being reused can be stored as variables in the root selector for better readability and usage.